app/app.go

```diff
@@ -1110,7 +1110,7 @@ func initParamsKeeper(
 	paramsKeeper.Subspace(ibchost.ModuleName)
 	paramsKeeper.Subspace(icahosttypes.SubModuleName)
 	// ethermint subspaces
-	paramsKeeper.Subspace(evmtypes.ModuleName).WithKeyTable(evmtypes.ParamKeyTable())
+	paramsKeeper.Subspace(evmtypes.ModuleName).WithKeyTable(evmtypes.ParamKeyTable()) //nolint: staticcheck
```

Comment:
The rest of all changes listed here is already on `main`, e.g. this line [here](https://github.com/evmos/evmos/blob/main/app/app.go#L1113) - not sure why it's still listed as a change